config/service-brokers/aiven/config.json

```diff
@@ -266,7 +266,7 @@
         },
         "plans": [
           {
-            "id": "f636ed93-3354-4173-b8bd-031f54866528",
+            "id": "f604ab6f-624a-4a7d-9849-4335e55408c4",
```

Comment:
I don't think, that's what you want here.

Reply:
Oh, weird. I don't remember doing that. Guess I wasn't paying attention. Really good catch!